.github/pull_request_template.md

```diff
@@ -0,0 +1,29 @@
+## Summary:
+- **What** did I do?
+- **Why** *** did I do it? (What was the motivation?)
+- **How** *** can someone test or demo my changes?
+
+  _*** **When relevant, use screenshots to document**_
+
+
+### Fixes or Addresses Issue \#: [ticket number & link]
+
+
+
+## Type of change:
+
+_Please delete options that are not relevant._
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
+
+
+## Checklist before requesting a review:
+- [ ] My code follows the guidelines of this project
+- [ ] I  performed a self-review of my code changes
+- [ ] I have completed spell-checking, removed unnecessary print statements & commented-out code
+- [ ] I have commented my code, particularly in hard-to-understand areas
+- [ ] I have made corresponding changes to the documentation
+- [ ] New and existing unit tests pass locally with my changes
```

Comment:
📓 : This checklist is great 🙌🏼 but probably not so relevant at this time right now since we haven't established a unit testing pattern / flushed out guidelines and there is a ton of commented out code already 😅. But going forward we can adhere to this to start establishing better code patterns